recipe/meta.yaml

```diff
@@ -1,45 +1,54 @@
 {% set name = "streamlit-aggrid" %}
-{% set version = "0.3.3" %}
+{% set version = "0.3.4" %}
```

Comment:
Should we go with `0.3.4.post3` or even `0.3.5`?

Reply:
`0.3.5` was yanked https://pypi.org/project/streamlit-aggrid/0.3.5/
I can do `0.3.4.post3`